src/openzaak/components/catalogi/management/commands/export.py

```diff
@@ -0,0 +1,96 @@
+import io
+import json
+import zipfile
+
+from django.apps import apps
+from django.core.management.base import BaseCommand, CommandError
+from django.http import HttpResponse
+from django.utils.translation import ugettext_lazy as _
+
+from rest_framework.test import APIRequestFactory
+from rest_framework.versioning import URLPathVersioning
+
+from openzaak.components.catalogi.api import serializers
+
+
+class Command(BaseCommand):
+    help = "Export the objects with the ids for the specified resource as json"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--archive_name", help=_("Name of the archive to write data to"), type=str
+        )
+        parser.add_argument(
+            "--response",
+            help=_("HttpResponse object to which the output data should be written"),
+            type=HttpResponse,
+        )
+        parser.add_argument(
+            "--resource",
+            action="append",
+            type=str,
+            help=_("Name of the resource to export objects for"),
+        )
+        parser.add_argument(
+            "--ids",
+            help=_("IDs of objects to be exported for the resource"),
+            action="append",
+            nargs="*",
+            type=int,
+        )
+
+    def handle(self, *args, **options):
+        archive_name = options.pop("archive_name")
+        response = options.pop("response")
+
+        if response and archive_name:
+            raise CommandError(
```

Comment:
for a future exercise, but just informing you - with argparse you can define mutually exclusive options. https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group